### ISSUE TITLE
[RCON]Commands returns to console, not to requester

### ISSUE THREAD

**Issue by ghost** [keepcalm/BukkitForge#483]
Plugin: None
Bukkitforge version: 229
Core minecraft function: RCon
Problem:
Any Rcon commands given to the server from a external source gets it response trown into console and not back to the Rcon requester.
as Rcon commands should not be shown in console only the:
[Minecraft] Rcon connection from: /0:0:0:0:0:0:0:1 (depending from which host ofcource)

This is not a issue most people will encounter but its blocking my development of my webpanel for modded minecraft servers

**Comment by ghost** [comment]
I am using buycraft which uses Rcon commands and they work great. All commands are printed on console but they get executed.

**Comment by ghost** [comment]
Yes it works, but i do not need it printed on console but returned to Rcon.

**Comment by various123** [comment]
As for 255 build I have same problem with rcon.

**Comment by ghost** [comment]
So coming back to this. Is this going to be fixed/solved?
